lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -828,17 +828,20 @@ def upload_grades_csv(_xmodule_instance_args, _entry_id, course_id, _task_input,
             course_grade.letter_grade,
             student.id in whitelisted_user_ids
         )
-        if certificate_info[0] == 'Y':
-            TASK_LOG.info(
-                u'Student is marked eligible_for_certificate'
-                u'(user=%s, course_id=%s, grade_percent=%s gradecutoffs=%s, allow_certificate=%s, is_whitelisted=%s)',
-                student,
-                course_id,
-                course_grade.percent,
-                course.grade_cutoffs,
-                student.profile.allow_certificate,
-                student.id in whitelisted_user_ids
-            )
+
+        TASK_LOG.info(
+            u'Student is marked eligible_for_certificate: %s '
```

Comment:
I figured it would be clear enough because the logged message will now include Y or N right after this statement, but I will update it to make it a bit more clear.